### PR TITLE
job feed screen

### PR DESCRIPTION
**Pull request by yongsokheng**

### CODE REVIEW

File: src/screen/job_feed/index.js [hunk 1]
```diff
@@ -1,29 +1,62 @@
...
 
 class JobFeedScreen extends Component {
+  componentDidMount() {
+    this.props.handleFetchData();
+  }
+
+  loadMore() {
+    if(!fetching) {
+      this.props.handleFetchData(this.props.data.page + 1, false, true);
+      fetching = true;
```
Comment (NgengChhengkim) [inline src/screen/job_feed/index.js:21]:
```
fetching = true;
this.props.handleFetchData(this.props.data.page + 1, false, true);
```

File: src/screen/job_feed/index.js [hunk 2]
```diff
@@ -1,29 +1,62 @@
...
-      <Container>
-        <Title title="Job feed"></Title>
-      </Container>
+      <AppContainer loading={this.props.data.loading}>
+        <List
+          data={this.props.data.listData}
+          renderItem={this.renderItem}
+          onRefresh={() => this.props.handleFetchData(1, true, false)}
+          onEndReached={() => this.loadMore()}
+          onMomentumScrollBegin={() => fetching = false}
```
Comment (NgengChhengkim) [inline src/screen/job_feed/index.js:47]: thu update state

Comment (yongsokheng) [reply src/screen/job_feed/index.js:47]: ko dc

File: src/screen/job_feed/index.js (hunk 1 above)
Comment (yongsokheng) [reply src/screen/job_feed/index.js:21]: thu roi, thay giong nhau thoi

**Review by NgengChhengkim — APPROVED** [review]

**Pull request merged**